### DISCUSSION TITLE
Show the date on which a repo was starred on, for each repo on the stars page

### DISCUSSION THREAD

**Discussion by robinmitra** [community/community#10287] [Other Feature Feedback, Questions, & Ideas]
Hello,

I have a lot of starred repo, like most people. When searching for a repo in my starred list, I often tend to browse based on my _rough_ memory of when I had starred the repo (by sorting the list by "recently starred"), and it generally works well.

I would like to suggest extending this further by also showing the date a repo was starred on, next to each repo (currently, only the date a repo was last updated is displayed). It can be further refined to _only_ show this date if a user has sorted the starred repos by "recently starred", so it won't be shown when sorted by "recently active" or "most stars".

This will help in the scenario where I can then quickly identify the repos that I had starred while I was working on "X Project" a couple of years ago, because I roughly remember the time frame during which I was working on that project. There are certainly other use-cases, but this one comes up fairly regularly for me.

It seems like a simply tweak, but not having this info limits the value provided by "recently starred" sorting order, especially if one has a large number of starred repos from the past.

Thanks,
Robin

**Comment by kizaski** [comment]
I second this

**Reply by Shai69bb** [reply]
I second this as well

**Reply by Rynkunas** [reply]
![ad8555ea052542d1c8addcfc76170452.png.jpg](https://github.com/community/community/assets/164784207/b5e4b7b4-16a6-404b-95fe-0c52ddc83099)

- [@robinmitra

**Reply by Rynkunas** [reply]
Неочень понимаю как все тут нада делать. Научите. Подскажите мня?
Спасибо

**Comment by Kirtjohn81** [comment]
> I second this

**Comment by carlosplanchon** [comment]
This is a very important feature to have.
This is a good feature in order to prove interest in a project. And this is useful in legal cases regarding licesing issues, in order to prove that the person showed interest in a project on a certain date.

**Comment by TPXP** [comment]
This is not shown in the web interface, but can be fetched from the API
```
curl -H "Accept: application/vnd.github.v3.star+json" https://api.github.com/users/$USER/starred | jq '.[] | select(.repo.full_name == "$ORG/$REPO") | .starred_at'
```
For example, to know when `torvalds` starred `torvalds/linux`:
```
curl -H "Accept: application/vnd.github.v3.star+json" https://api.github.com/users/torvalds/starred | jq '.[] | select(.repo.full_name == "torvalds/linux") | .starred_at'
"2011-09-04T22:48:12Z"
```
